x-pack/plugins/observability_solution/observability/public/utils/test_helper.tsx

```diff
@@ -52,7 +51,7 @@ const queryClient = new QueryClient({
 
 export const render = (component: React.ReactNode, config: Subset<ConfigSchema> = {}) => {
   return testLibRender(
-    <IntlProvider locale="en-US" messages={translations.messages}>
+    <IntlProvider locale="en-US">
```

Comment:
What is this change about?

Reply:
Removing dependency towards `@kbn/translations-plugin` (which is private).
I've checked with @Bamieh and apparently there is no need to pass this object.